packages/ui-identicon/src/index.ts

```diff
@@ -10,7 +10,11 @@ import newShape from './shape/circle';
 import newElement from './svg/element';
 import { SHAPE_COUNT } from './defaults';
 
-export default function identicon (seed: string | Uint8Array, diameter: number = 256, className: string = '', style?: { [index: string]: string }): HTMLElement {
+export default function identicon (seed: string | null, diameter: number = 256, className: string = '', style?: { [index: string]: string }): HTMLElement {
+  if (!seed) {
+    throw new Error('Unable to generate identicon from empty seed');
```

Comment:
Was a bit sceptical, however we _should_ now never get here since we have the traps on the UI components. (And this happens more that we would like, i.e. creating new account, it just calls without regard for validity)

TL;DR Makes sense with the UI traps